test/functional/rpc_packages.py

```diff
@@ -340,6 +343,13 @@ def test_submitpackage(self):
         assert_raises_rpc_error(-25, "package topology disallowed", node.submitpackage, chain_hex)
         assert_equal(legacy_pool, node.getrawmempool())
 
+        assert_raises_rpc_error(-8, f"Array must contain between 1 and {MAX_PACKAGE_COUNT} transactions.", node.submitpackage, [chain_hex[0]] * 0)
```

Comment:
found it weird
```Suggestion
        assert_raises_rpc_error(-8, f"Array must contain between 1 and {MAX_PACKAGE_COUNT} transactions.", node.submitpackage, [])
```

Reply:
I quite liked the symmetry/progression of the `* 0` notation but I don't really care either way and agree that `[]` is readable as well, so addressed in force push.